.github/ISSUE_TEMPLATE/new_python_tool_pip.yml

```
@@ -0,0 +1,119 @@
+name: 🆕 NEW PYTHON PACKAGE from PyPI
+description: Propose the addition of a new Python package/tool that is installed with pip
+title: "Python package proposal: <package_name>.vm"
```

We can install Python tools in a different way than with pip, I would use pip explicetly:

```suggestion
title: "pip package proposal: <package_name>.vm"
```